### ISSUE TITLE
Make inbox filters links

### ISSUE THREAD

**Issue by mxstbr** [withspectrum/spectrum#4556]
- [x] Feature request

https://github.com/withspectrum/spectrum/blob/9bb7010cd0f22d4a59e1a74f185cdc4a78ef1b28/src/views/dashboard/components/communityList.js#L110-L139

This whole bit should be a link that redirects to `/communityslug` when the JS hasn't rehydrated.

**Comment by srikar0896** [comment]
@mxstbr can I pick this up?

**Comment by mxstbr** [comment]
Yes please @srikar0896!

**Comment by srikar0896** [comment]
Hi @mxstbr, It's my first pr, I am trying to get aware of codebase. It's a bit unclear. What do you mean by redirect to `/communityslug` **when the JS hasn't rehydrated**? What is be the expected behaviour?

**Comment by srikar0896** [comment]
Okay, I want to get some context here. So If we are simply redirecting the user to `https://spectrum.chat/${community.slug}` onclick why do we need `onClick` handler. So do we want to simply redirect the user to `https://spectrum.chat/${community.slug}` onClick or do something and then redirect in `onClick` handler.

**Comment by mxstbr** [comment]
We only want the user to redirect if the JavaScript has not rehydrated yet. Otherwise, if you click on it, nothing happens.

When the JS has rehydrated, we do not want to redirect, we want to filter the inbox by that community, so we let the `onClick` handler do its thing!

Does that make sense? Sorry for not explaining it well :pray:

**Comment by mxstbr** [comment]
> no matter what it does in onClick handler it always redirects to /communitySlug right.

I think this is not true—the `onClick` handler only needs to call `evt.preventDefault()` and the `href` will not be opened I'm pretty sure!

So we don't need to know whether JS has rehydrated or not, we only need to call `evt.preventDefault()` from the `onClick` and we should be good.

**Comment by srikar0896** [comment]
@mxstbr yes, that makes sense. But If we wrap the `communityListItem` with `Link` we don't even need to call `e.preventDefault()` in `onClick` handler. Please review https://github.com/withspectrum/spectrum/pull/4565
